include/xrpl/protocol/detail/b58_utils.h

```diff
@@ -169,8 +169,9 @@ inplace_bigint_div_rem(std::span<uint64_t> numerator, std::uint64_t divisor)
 [[nodiscard]] inline std::array<std::uint8_t, 10>
 b58_10_to_b58_be(std::uint64_t input)
 {
-    constexpr std::uint64_t B_58_10 = 430804206899405824;  // 58^10;
+    static constexpr std::uint64_t B_58_10 = 430804206899405824;  // 58^10;
     assert(input < B_58_10);
+    (void)B_58_10;
```

Comment:
What does this line do?

Reply:
It fools the compiler into thinking the variable is used, so you don't get unused variable warnings when `assert`s are disabled.
```
In file included from /home/extra/ed/dev/rippled/current/src/ripple/protocol/impl/tokens.cpp:32:
/home/extra/ed/dev/rippled/current/src/ripple/protocol/impl/b58_utils.h: In function ‘std::array<unsigned char, 10> ripple::b58_fast::detail::b58_10_to_b58_be(uint64_t)’:
/home/extra/ed/dev/rippled/current/src/ripple/protocol/impl/b58_utils.h:172:29: warning: unused variable ‘B_58_10’ [-Wunused-variable]
  172 |     constexpr std::uint64_t B_58_10 = 430804206899405824;  // 58^10;
      |                             ^~~~~~~
```
It's just an annoyance, but it was on the branch I had sitting around, so I included it.

Reply:
Yeah, I keep forgetting that exists. LOL